go/obscuronode/enclave/enclaverunner/enclave_runner.go

```diff
@@ -41,7 +40,7 @@ func SetLogs(writeToLogs bool, logPath string) {
 	if writeToLogs {
 		logFile, err = os.Create(logPath)
 		if err != nil {
-			panic(err)
+			panic(fmt.Sprintf("could not create log file. Cause: %s", err))
```

Comment:
here

Reply:
We can't log anything here as the problem is we can't create a log file.

Reply:
log is always available. It's fine :)

Reply:
But here's we're literally saying we can't find the file we want to log to. So how can we possibly log?

Reply:
It logs to stdout